doc/userguide/src/ExecutingTestCases/TestExecution.rst

```diff
@@ -364,6 +364,85 @@ converting any failure into a continuable failure. These failures are
 handled by the framework exactly the same way as continuable failures
 originating from library keywords.
 
+Controlling Continue on Failure Using Reserved Tags
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
We haven't used title case with other titles so this should also be spelled like "Controlling continue on failure ...".

As a non-native English speaker I'm not sure is "continue on failure" better than "continue-on-failure" but for some reason the latter "feels" better for me. Should check what style has been used elsewhere and use one style consistently (which can mean updating other docs).

Reply:
We do call it `Continue on failure` mode, including a section title.. I checked with a native english speaker, and he also recommend to stick without dashes. so would suggest we keep it as is.